install/upgrade/update_to_dev.php

```diff
@@ -224,6 +224,9 @@ function plugin_flyvemdm_update_to_dev(Migration $migration) {
    $migration->addfield($table, 'plugin_orion_tasks_id', 'integer', ['after' => 'dl_filename']);
    $migration->changeField($table, 'name', 'package_name', 'string');
    $migration->migrationOneTable($table);
+   if ($DB->fieldExists($table, 'filesize')) {
```

Comment:
The object Migration already handles the possible non existence of the column

Reply:
updated on last commit